bundles/statistics/statsgrid2016/instance.js

```diff
@@ -89,6 +89,27 @@ Oskari.clazz.define(
 
             // regionsetViewer creation need be there because of start order
             this.regionsetViewer = Oskari.clazz.create('Oskari.statistics.statsgrid.RegionsetViewer', this, sandbox, this.conf);
+
+            // Crete indicators tab to personal data view if personaldata bundle exists
+            var reqName = 'PersonalData.AddTabRequest';
+            if (sandbox.hasHandler(reqName)) {
+                me._addIndicatorsTabToPersonalData(sandbox);
+            } else {
+                // Wait for the application to load all bundles and try again
+                Oskari.on('app.start', function (details) {
+                    if (sandbox.hasHandler(reqName)) {
+                        me._addIndicatorsTabToPersonalData(sandbox);
+                    }
+                });
+            }
+        },
+        _addIndicatorsTabToPersonalData: function (sandbox) {
+            var tab = Oskari.clazz.create('Oskari.statistics.statsgrid.MyIndicatorsTab', this);
+            tab.bindEvents();
+            var addAsFirstTab = false;
+            var reqBuilder = sandbox.getRequestBuilder('PersonalData.AddTabRequest');
+            var req = reqBuilder(tab.getTitle(), tab.getContent(), addAsFirstTab, tab.getId());
```

Comment:
Use Oskari.requestBuilder() instead and check if you got a function as return value. Statsgrid is used in embedded maps as well and those don't usually have personaldata bundle so the request won't be available.